### DISCUSSION TITLE
changing colors by button problem

### DISCUSSION THREAD

**Discussion by Diddlybabble** [nesbox/TIC-80#2239] [Q&A]
ay mates i wanted to change the screen color by pressing a button but it doesn't work anyways here is it code :color=0
function TIC()
cls(color)
if btnp(4) then
color = color+1
elseif btnp (13) then
   color = color-1
  end 
end

**Comment by nesbox** [comment]
Tyr this
``` lua
color=0
function TIC()
cls(color)
if btnp(2) then
color = color+1
elseif btnp (3) then
color = color-1
end
end
```
